src/proto/grpc/testing/messages.proto

```diff
@@ -309,3 +317,23 @@ message TestOrcaReport {
   map<string, double> request_cost = 3;
   map<string, double> utilization = 4;
 }
+
+enum HookRequestCommand {
```

Comment:
I don't see these being used

Reply:
They are from that other PR. This file source of truth is https://github.com/grpc/grpc-proto/blob/master/grpc/testing/messages.proto that already has the messages.

Reply:
can you merge master then please?

Reply:
Rebased on top of "hook server" change, should be easier to follow now.